python/adelphi/adelphi/__init__.py

```diff
@@ -0,0 +1,10 @@
+# list of system keyspaces to filter out
+system_keyspaces = ["system",
+                    "system_schema",
+                    "system_traces",
+                    "system_auth",
+                    "system_distributed",
+                    "system_virtual_schema",
+                    "system_views"]
```

Comment:
I'm not really sure this is the right place for this logic... but I couldn't find another natural place to put it.  Perhaps it should go into store.py since that's where it's actually used... that might make the most sense.

Reply:
Subsequently moved into adelphi.store